pom.xml

```diff
@@ -0,0 +1,145 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.logging.log4j</groupId>
+    <version>2.18.0</version>
+    <artifactId>azure-kusto-log4j</artifactId>
+    <name>Apache Log4j Kusto</name>
+    <description>Kusto appender for Log4j.</description>
+    <properties>
+        <docLabel>Kusto Documentation</docLabel>
+        <projectDir>/azure-kusto-log4j</projectDir>
+        <module.name>org.apache.logging.log4j.kusto</module.name>
+        <maven.doap.skip>true</maven.doap.skip>
+        <awaitility.version>4.2.0</awaitility.version>

```

Comment:
Use empty lines and comments to create distinct blocks. For example, version number properties should be in their own distinct block.

Reply:
Reminder